packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -209,7 +209,7 @@
 {/each}
 
 <HiddenInPrint enabled={hideDuringPrint}>
-	<div class="mt-2 mb-4 ml-0 mr-2 inline-block">
+	<div class="mt-2 mb-4 ml-0 mr-2 inline-block" data-testid="Dropdown-{name}">
```

Comment:
Instead of adding a `data-testid` here, we should add an `aria-label` to the dropdown trigger (`<Button />`) so that it is accessible

```jsx
<Button
  role="combobox"
  aria-label={title ?? formatTitle(name)}
/>
```

Then we can query this dropdown like this:
```js
page.getByRole('combobox', { name: '<dropdown title or name>' })
```